src/components/picture/_children/source.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from "prop-types";
+import formatSrcset from "../../../utils/format-srcset";
+
+const Source = ({ height, media, resizerOptions, resizerURL, src, width, ...rest }) => {
+	if (media && src) {
+		if (resizerOptions && resizerOptions.auth) {
+			return (
+				<source
+					{...rest}
+					className="c-picture__source"
+					height={height}
+					media={media}
+					srcSet={formatSrcset(resizerURL.concat(src), resizerOptions, width, height)}
+					width={width}
+				/>
+			);
+		}
+		return <source {...rest} className="c-picture__source" media={media} srcSet={src} />;
```

Comment:
```suggestion
		return <source {...rest} media={media} srcSet={src} />;
```
There should never be a reason to style the source... it's also not tokenized.